packages/components/atoms/f-button/test-utils/component-objects/f-button--link.component.js

```diff
@@ -1,6 +1,6 @@
 const Page = require('@justeat/f-wdio-utils/src/page.object');
 
-module.exports = class Buttons extends Page {
+module.exports = class LinkButtons extends Page {
```

Comment:
Should the name be a plural?

Reply:
I was wondering this too but this class goes to the story file with multiple buttons on the page, so I think it's okay to leave as is

Reply:
changed my mind and renamed these actually since they're always imported as a singular in the tests 👍